Makefile

```diff
@@ -150,7 +150,7 @@ export APIGEE_ENVIRONMENT=$(env)
 export APIGEE_ACCESS_TOKEN=$(token)
 
 space :=
-space += 
+space +=
```

Comment:
Dodgy merge conflict resolution?

Reply:
Second space should be there and be equal to " ", it's a makefile workaround to remove a space from the username variable, cannot set soace variable to " " itself but you can append a space to an existing blank variable. Sure there's a better way of achieving this

Reply:
Nope it's intellij being helpful

Reply:
@alexmccormick-IW github didn't feel like showing me your message. I knew it was supposed to be there, the trouble was stopping intellij from removing it every time we commit :joy: Found another hacky way of defining a space instead